cmd/tink-cli/cmd/workflow/list.go

```diff
@@ -27,6 +27,17 @@ var listCmd = &cobra.Command{
 	Use:     "list",
 	Short:   "list all workflows",
 	Example: "tink workflow list",
+	Deprecated: `This command is deprecated and it will change at some
+	point. Please try what follows:
+
+	# If you want to retrieve a single workflow you know by ID
+	tink workflow get [id]
+	# You can print it in JSON and CSV as well
+	tink workflow get -o json [id]
+
+	# Get a list of available workflows
+	tink workflow get [id]
```

Comment:
This should just point users to run `tink workflow get` instead or maybe we can print the example somehow?